rasa/utils/tensorflow/model_data.py

```diff
@@ -152,7 +157,7 @@ def _validate_number_of_dimensions(
                 f"array: {input_array}."
             )
 
-    def get_shape_type_info(self) -> Tuple[List[Tuple], List[Tuple]]:
+    def get_shape_type_info(self) -> Tuple[List, List]:
```

Comment:
do we know what these `List`s contain?

Reply:
It depends on the array. Sometimes it is a list of tuples of ints, sometimes it is just an int. I could change it to
```
Tuple[List[Union[int, Tuple[None], Tuple[None, int], Tuple[None, None, int], Tuple[None, None, None, int]]], List[int]]
```

Reply:
oh no